packages/firebase/compat/index.d.ts

```diff
@@ -1315,7 +1315,7 @@ declare namespace firebase {
   /**
    * Gets the {@link firebase.remoteConfig.RemoteConfig `RemoteConfig`} instance.
    *
-   * @webonly
+   * This is not available in Node.js.
```

Comment:
Is there a reason not to use the same wording as for Auth functions, "This method is not available in Node.js?"

Maybe in some cases these are not methods; but to be strictly consistent, we might want to say "this property/class/thing is not available in Node.js."

Reply:
Honestly, I searched and replaced webonly in this file and was too lazy to make custom sentences for each. I could do that, I think it's less than 20 instances.

Reply:
Updated.